value_test.go

```diff
@@ -68,3 +69,37 @@ func TestIP(t *testing.T) {
 		}
 	}
 }
+
+func TestRedactURL(t *testing.T) {
```

Comment:
Can we stick with IP example?

Reply:
I'm the last person to say not to write a test but honestly I do not feel it needs much testing, the example is good enough.